voxpupuli-puppet-lint-plugins.gemspec

```
@@ -15,26 +15,26 @@ Gem::Specification.new do |s|
 
   # pull in puppet-lint 2.5.0; required for Ruby 3 support
   s.add_runtime_dependency 'puppet-lint', '>= 2.5.0'
-  s.add_runtime_dependency 'puppet-lint-absolute_classname-check', '>= 2.0.0'
-  s.add_runtime_dependency 'puppet-lint-anchor-check'
-  s.add_runtime_dependency 'puppet-lint-classes_and_types_beginning_with_digits-check'
-  s.add_runtime_dependency 'puppet-lint-file_ensure-check'
-  s.add_runtime_dependency 'puppet-lint-leading_zero-check'
-  s.add_runtime_dependency 'puppet-lint-legacy_facts-check'
-  s.add_runtime_dependency 'puppet-lint-lookup_in_parameter-check'
-  s.add_runtime_dependency 'puppet-lint-manifest_whitespace-check'
-  s.add_runtime_dependency 'puppet-lint-optional_default-check'
-  s.add_runtime_dependency 'puppet-lint-param-docs'
-  s.add_runtime_dependency 'puppet-lint-params_empty_string-check'
-  s.add_runtime_dependency 'puppet-lint-param-types'
-  s.add_runtime_dependency 'puppet-lint-resource_reference_syntax'
-  s.add_runtime_dependency 'puppet-lint-strict_indent-check'
-  s.add_runtime_dependency 'puppet-lint-top_scope_facts-check'
-  s.add_runtime_dependency 'puppet-lint-topscope-variable-check'
-  s.add_runtime_dependency 'puppet-lint-trailing_comma-check'
-  s.add_runtime_dependency 'puppet-lint-unquoted_string-check'
-  s.add_runtime_dependency 'puppet-lint-variable_contains_upcase'
-  s.add_runtime_dependency 'puppet-lint-version_comparison-check'
+  s.add_runtime_dependency 'puppet-lint-absolute_classname-check', '>= 3.0.1', '< 4.0.0'
+  s.add_runtime_dependency 'puppet-lint-anchor-check', '>= 1.0.1', '< 2.0.0'
+  s.add_runtime_dependency 'puppet-lint-classes_and_types_beginning_with_digits-check', '~> 1'
+  s.add_runtime_dependency 'puppet-lint-file_ensure-check', '~> 1'
+  s.add_runtime_dependency 'puppet-lint-leading_zero-check', '~> 1'
+  s.add_runtime_dependency 'puppet-lint-legacy_facts-check', '>= 1.0.4', '< 2.0.0'
+  s.add_runtime_dependency 'puppet-lint-lookup_in_parameter-check', '~> 1'
+  s.add_runtime_dependency 'puppet-lint-manifest_whitespace-check', '>= 0.2.6', '< 1.0.0'
```

Semver says that on a 0.x we should treat 0.1.z -> 0.2.0 as a major version:
```suggestion
  s.add_runtime_dependency 'puppet-lint-manifest_whitespace-check', '~> 0.2.6'
```